frontend/packages/topology/src/filters/KindFilterDropdown.tsx

```diff
@@ -81,7 +89,7 @@ const KindFilterDropdown: React.FC<KindFilterDropdownProps> = ({
           data-test={filter.label}
         >
           <ResourceIcon kind={filter.id} />
-          {filter.label} ({supportedKinds[filter.id]})
+          {filter.labelKey} ({supportedKinds[filter.id]})
```

Comment:
Keep label here, so that it doesn't look like a bug that you don't have a `t()`  call here. Two lines above the `filter.label` might be replaced with filter["data-test"] when this was the reason that you doesn't changed `filter.label` directly.

```suggestion
          {filter.label} ({supportedKinds[filter.id]})
```